analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.cs

```diff
@@ -3532,3 +3532,32 @@ public void TestMethod()
 
     private void Callback() { }
 }
+
+// Reproducer for https://github.com/SonarSource/sonar-dotnet/issues/8484
```

Comment:
For simplicity, the class name is clear enough about the fact that it's a repro
```suggestion
// https://github.com/SonarSource/sonar-dotnet/issues/8484
```